sal_interfaces/MTMount/MTMount_Events.xml

```diff
@@ -1,85 +1,245 @@
-    <item>
-      <EFDB_Name>id</EFDB_Name>
-      <Description>Identifier of the error</Description>
+    <EFDB_Topic>MTMount_logevent_error</EFDB_Topic>
+    <item>
+      <EFDB_Name>latched</EFDB_Name>
+      <Description>Is the error condition latched on (as it been active since it was last reset)?</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>active</EFDB_Name>
+      <Description>Is the error condition present?</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>code</EFDB_Name>
+      <Description>Error identifier code.</Description>
       <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>text</EFDB_Name>
-      <Description>The error info</Description>
+      <Description>Text description of error.</Description>
       <IDL_Type>string</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>subsystem</EFDB_Name>
+      <Description>Subsystem that produced the error; a single Subsystem enumeration value.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>MTMount</Subsystem>
+    <EFDB_Topic>MTMount_logevent_target</EFDB_Topic>
+    <Description>Commanded position and velocity.</Description>
+    <item>
+      <EFDB_Name>elevation</EFDB_Name>
+      <Description>Target mount elevation angle at the specified time.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>elevationVelocity</EFDB_Name>
+      <Description>Target mount elevation angle velocity at the specified time.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg/second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>azimuth</EFDB_Name>
+      <Description>Target mount azimuth angle at the specified time. The allowed range is 0 to 360.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>azimuthVelocity</EFDB_Name>
+      <Description>Target mount azimuth velocity at the specified time.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg/second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>taiTime</EFDB_Name>
+      <Description>TAI date at which the commanded positions and velocities apply, as unix time.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>trackId</EFDB_Name>
+      <Description>Unique target identifier </Description>
```

Comment:
Please replace the trailing space with a `.`

Reply:
Done (as part of the previous change).